### ISSUE TITLE
Update labels: Issue Template: gha-issue-template

### ISSUE THREAD

**Issue by cmedina-dev** [hackforla/website#4475]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.
### Overview
As developers, we need to add the labels from the [Missing series](https://github.com/hackforla/website/wiki/How-to-read-and-interpret-labels#missing-series) so that we can avoid the GitHub bot taking off the labels when devs add them to issues.
### Action Items
- [x] Navigate to `.github/ISSUE_TEMPLATE/gha-issue-template.yml`
- [x] Add the following line under the description field
   ```
   labels: ['role missing', 'Complexity: Missing', 'Feature: Board/GitHub Maintenance', 'size: missing','Draft']
   ```
- [x] Test the updated template using these [instructions](https://github.com/hackforla/website/issues/4307#issuecomment-1549888931).  Copy the contents of that comment into a new comment in this issue, and check off the boxes as you complete the steps.  

**For PR Reviewers and Merge Team**
To review this issue, click the link below under "Link for reviewers" and verify that the labels match the updated labels string in the issue.

**Link for Reviewers**
This URL will be inserted by the dev during the test process and will be used by reviewers to preview the updated template.
URL of the issue branch on the test Repository:  [REPLACE THIS TEXT AND BRACKETS WITH THE URL]

**For PM, Merge Team, or Tech Lead**
- [ ] Once the pull request associated with this issue is approved and merged, please update and edit epic #4307 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `.github/ISSUE_TEMPLATE/gha-issue-template.yml`
- This issue is part of epic #4307

**Comment by vanyanv** [comment]
Availability: M-F 10-8 pm;
ETA: 05/19/23

**Comment by roslynwythe** [comment]
Hi @vanyanv I added a final Action Item to describe the process for testing the updated issue template.

**Comment by roslynwythe** [comment]
Hi @vanyanv I had to update the test instructions.  Please let me know if you have any questions during test.  Thanks!

**Comment by roslynwythe** [comment]
Hi @vanyanv this issue has been flagged for review by dev leads, as it has been inactive for at least 2 weeks.  Please leave an update using the format outlined in the comment above, and don't forget to remove the '2 weeks inactive' label and add the 'Status: Updated` label.  You are also welcome to send me a Slack DM if you need assistance with this issue.  Thank you.

**Comment by ExperimentsInHonesty** [comment]
@vanyanv we are removing you from this issue for lack of communication.

**Comment by one2code** [comment]
**Availability:** 6/27/23 8:45-9:30 PM EST, 11:45-12:30 am, 6/29/23 9 -10PM EST, 6/30/23 7-10 PM  
**ETA:** 6/30/23

**Comment by one2code** [comment]
### Instructions for testing updated issue templates:
- [x] Push your issue branch to your fork of the website repository, as described in [Section 3.1.a of CONTRIBUTING doc](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#31a-push-all-changes-to-your-issue-branch), for example:  `git push --set-upstream origin test-update-labels-1234`  
- [x] The response from the push command will include the URL for creating a pull request.  Copy/Paste the URL below for use after testing:  
 https://github.com/one2code/website/pull/new/gha-issue-template-4475
- [x] Browse to your fork of the website repository and click Settings 
<details><summary>Screenshot of Repository Settings</summary>

![image](https://github.com/hackforla/website/assets/5314153/8ecf2e55-2187-4ab9-acf1-2fd7548291a1)

</details> 

- [x] Change the default branch to the new issue branch

<details><summary>Screenshot of Default Branch Setting</summary>

![image](https://github.com/hackforla/website/assets/5314153/945fe02e-035f-4215-9c25-1d46c209d8c0)

</details> 

- [x] Further down on the settings page under Features, check the Issues checkbox

<details><summary>Screenshot of Issues Checkbox</summary>

![image](https://github.com/hackforla/website/assets/5314153/945fe02e-035f-4215-9c25-1d46c209d8c0)

</details> 

- [x] Browse to the Issues page, then click Labels

<details><summary>Screenshot - creating labels in the test Repo</summary>

![image](https://github.com/hackforla/website/assets/5314153/208e1b07-2478-4ed3-98a1-207de58d25c5)

</details> 

- [x] Create labels corresponding to each label that was listed in the updated issue template.  The color of the labels don't have to match those on the `hackforla/website` repository, but the text of the labels must match exactly.  
- [x] From the Issues page, click `New Issue` then locate the template updated by your issue, and click `Get Started` to open the issue template. 
- [x] Copy/Paste the URL of the issue template into the Resources section of the issue, replacing the text: https://github.com/one2code/website/issues/new?assignees=&labels=role+missing%2CComplexity%3A+Missing%2CFeature+Missing%2Csize%3A+missing%2CDraft&projects=&template=gha-issue-template.yml&title=Sample+form+for+GitHub+actions+
 This will enable others to check the template.  
- [x] Verify the appearance of the issue template, including careful examination of the labels.  If there is a discrepancy between the labels listed in the issue and the labels that appear on the issue template, ask a dev lead or merge team member for help.  
- [x] Complete the PR process by visiting the URL saved from the push command
- [x] Copy these sections from the Issue into the PR 
  - [ ] "For PR Reviewers and Merge Team"
  - [ ] "Link for Reviewers"
  - [ ] "For PM, Merge Team, or Tech Lead"
